src/runtime/getUrl.js

```diff
@@ -21,7 +21,7 @@ module.exports = (url, options) => {
   // Should url be wrapped?
   // See https://drafts.csswg.org/css-values-3/#urls
   if (/["'() \t\n]|(%20)/.test(url) || options.needQuotes) {
```

Comment:
Why we need escape `\\` in runtime? We don't touch such stuff at all, even more, it can literally break already escaped URL

Sorry, we don't need this changes here, even more it worsens the performance

Reply:
@alexander-akait we are not escaping the \\ in runtime. we are replacing single \ with \\ to escape the URL. \\ is there in the code to escape a single \. Otherwise JavaScript consider single \ as an escaping character itself.

Reply:
And what could be the problem? Please provide a **real scenario** that could be used in `url()` and CSS and put it there - https://github.com/webpack-contrib/css-loader/blob/master/test/fixtures/url/url.css, I wouldn't want us to waste perf to something that can never really happen,

Reply:
@alexander-akait what if developers have css like this? Does it work right now? I don't think so as the current code doesn't escape the backslashes

`.backslashes {
  background: url("https://www.example.com?path=path\to\resource");
}`

Reply:
How does this violate security?

Reply:
@ruchira-net Some of them is ident some of them is string, and depending on this we should escaping them or not, I don't see a real integration test of the problem, so I asked to put it in the integration tests and not the unit, I don't want to degrade performance because there is no real problem